.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Python Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "python.defaultInterpreterPath": "/usr/bin/python3.12"
+            },
+            "extensions": [
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "ms-python.debugpy",
+                "ms-azuretools.vscode-docker",
+                "mechatroner.rainbow-csv"
+            ]
+        }
+    },
+    "postCreateCommand": "poetry install",
```

Comment:
@GalenReich This is my favorite part about this. Once anyone creates the dev container, they don't even need to run `poetry install`. They have it all configured for them! I just created a new Codespace and ran `poetry run pytest` as my first command and it worked as expected. 

I love Dev Containers. I don't get to use them at my job but I wish I could (we have something similar but it's less configurable).